v2/emailpassword/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -4,20 +4,72 @@ title: 2. Backend config
 hide_title: true
 ---
 
+<!-- COPY SECTION -->
+<!-- ./emailpassword/serverless/with-aws-lambda/backend-config.mdx -->
+<!-- 1 -->
+
 import AppInfoForm from "/src/components/appInfoForm"
 import CoreInjector from "/src/components/coreInjector"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
 
 # 2. Backend config
 
-## 1) Install supertokens package
+## 1) Create Lambda layer for required libraries
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
 ```bash
-npm i supertokens-node
+mkdir lambda && cd lambda
+touch lambdaFunction.js
+
+# Run these commands
+npm i -s supertokens-node @middy/core @middy/http-cors
+cp -r node_modules nodejs # TODO: Verify this
```

Comment:
either verify this, or remove the TODO comment

Reply:
Removed.